### ISSUE TITLE
buffer_sizeof argument 1 invalid reference to (buffer) - requested 12 max is 0

### ISSUE THREAD

**Issue by time-killer-games** [YoYoGames/GameMaker-Bugs#2316]
### Description

```
___________________________________________
############################################################################################
ERROR in
action number 1
of Create Event
for object obj_example:

buffer_sizeof argument 1 invalid reference to (buffer) - requested 12 max is 0
 at gml_Object_obj_example_Create_0 (line 4) - chan = buffer_sizeof(buffer_u64);
############################################################################################
gml_Object_obj_example_Create_0 (line 4)
```

Note the ticket says doesn't work on mac or ubuntu as well though that is just a guess. technically only been verified on Windows so far.

Contact Us Package:
https://drive.google.com/file/d/1TNSOcmZ8AOaeA3weazUzUC28If0bSnGG/view?usp=sharing

### Expected Change

I'd like to see buffers working again. This project works in Monthly Version 2023.8.2.108 with default runtime. Shows above error in Beta Version 2023.1100.0.444 with default beta runtime

### Steps To Reproduce

1. Start GameMaker
2. [Download my WebMPlayer.zip Example Project from here (load in GameMaker and hit run)](https://samuel-venable.itch.io/gamemaker-extension-collection)
3. Test with older runtimes to see the desired outcome without errors

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

Beta (Please put the exact version number in your Description)

### Which platform(s) are you seeing the problem on?

Windows, macOS, Ubuntu

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [X] I have included a small sample project

**Comment by YYDan** [comment]
Confirmed the repro steps in IDE v2023.1100.0.450 Beta Runtime v2023.1100.0.455.

Issue is now fixed in  IDE v2023.1100.0.450  Runtime v2023.1100.0.459
![image](https://github.com/YoYoGames/GameMaker-Bugs/assets/15669932/d98dfa2a-0259-4af9-ae80-771afee05da7)

Verifying.

**Comment by time-killer-games** [comment]
Thank you so very much of handling this so quickly! You guys rock! :D
